view/frontend/web/js/model/checkout.js

```diff
@@ -174,70 +174,28 @@ function(
       var rowsToHide = [];
 
       Model.rates().forEach(function(rate) {
-        var rows = Model.getShippingMethodRows(rate.method_code);
+        const hasDeliveryOptions = Model.hasDeliveryOptions();
```

Comment:
var->const is wel spannend vanwege browser compatibility. IE ligt er natuurlijk wel al een tijdje uit... Dit zou je _uiteindelijk_ ook voor de hele file kunnen doen, ff een slagje moderniseren